library/CMakeLists.txt

```diff
@@ -69,12 +69,10 @@ set(MAIN_HEADERS
     include/Memory.h
     include/MiscUtils.h
     include/Module.h
-    include/Pragma.h
```

Comment:
oh good, I'm glad you got both of them

Reply:
i totally didn't notice this; i just had a search for "Pragma.h" in VS code and was just deleting things until there weren't any hits left :)